.gitlab-ci.yml

```diff
@@ -23,48 +25,70 @@ variables:
     kubectl config set-cluster cluster --server="$KUBE_URL" --certificate-authority="$HOME/ca.pem" &&
     kubectl config set-credentials cluster --token="$KUBE_TOKEN" && kubectl config set-context cluster --cluster=cluster --user=cluster --namespace="$KUBE_NAMESPACE" &&
     kubectl config use-context cluster
+  - &KUBEAPPLY
+    kubectl apply -f <(envsubst <etc/kube/service_template.yaml) &&
+    kubectl apply -f <(envsubst <etc/kube/deployment_template.yaml) &&
+    kubectl rollout status -f <(envsubst <etc/kube/deployment_template.yaml)
 
 build:
   stage: build
-  only:
-    - master
-    - prod
   script:
     # Clear *_TOKEN variables during code build so that compile-time code can't access them
     - CI_BUILD_TOKEN="" KUBE_TOKEN="" PROD_KUBE_TOKEN="" PROD_DOCKER_PASSWORD="" etc/scripts/stage_docker.sh --install-ghc --test
-    - docker build -t "${DOCKER_IMAGE}" etc/docker
+    - docker build -t "${DEPLOYMENT_IMAGE}" etc/docker
     - docker login -u gitlab-ci-token -p "${CI_BUILD_TOKEN}" "${CI_REGISTRY}"
-    - docker push "${DOCKER_IMAGE}"
+    - docker push "${DEPLOYMENT_IMAGE}"
     - |
       if [[ "$CI_BUILD_REF_NAME" == "master" ]]; then
-        docker tag "${DOCKER_IMAGE}" "${CI_REGISTRY_IMAGE}:latest"
+        docker tag "${DEPLOYMENT_IMAGE}" "${CI_REGISTRY_IMAGE}:latest"
         docker push "${CI_REGISTRY_IMAGE}:latest"
       fi
 
-deploy_master:
+deploy_review:
   stage: deploy
   only:
+    - branches
+  except:
     - master
+    - prod
```

Comment:
Probably don't need to exclude the `prod` branch anymore, since it's no longer being used for deployments to production.  I noticed that `fpcomplete-site`'s `.gitlab-ci.yml` also excludes it; most likely an oversight on my part when I modified it to deploy `master` to the production.